kratos/python/add_space_to_python.cpp

```diff
@@ -0,0 +1,50 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Vicente Mataix Ferrandiz
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/define_python.h"
+#include "python/add_space_to_python.h"
+#include "spaces/ublas_space.h"
+
+namespace Kratos::Python
+{
+void AddSpaceToPython(pybind11::module& m)
```

Comment:
I am 99% sure this exists already somewhere 

Probably in the addstrategiestopython

Having it separate makes sense but not duplicated

Reply:
A priori it is not, at least I couldn't find it. UblasSpace is old...